docs/source/conf.py

```diff
@@ -0,0 +1,159 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+from pkg_resources import get_distribution
+
+from m2r import MdInclude
+from recommonmark.transform import AutoStructify
+
+# -- Project information -----------------------------------------------------
+
+project = 'labscript-utils'
+copyright = '2020, labscript suite'
+author = 'labscript suite contributors'
+
+
+# The full version, including alpha/beta/rc tags
+version = get_distribution(project).version
+release = version
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.napoleon",
+    # "sphinx.ext.linkcode",
+    "sphinx.ext.viewcode",
+    "sphinx_rtd_theme",
+    "recommonmark",
+]
+
+
+#
+# This code is for sphinx.ext.linkcode to link to GitHub source directly. 
```

Comment:
Is the intention to make this operational in a future PR?

Reply:
I was going to ask what you thought. Linking to GitHub source seems nice, but also the feature just links to the file, not the source code line (so is not as good as `viewcode`).

Reply:
I've removed this for now